src/util/arith_tools.h

```diff
@@ -10,12 +10,13 @@ Author: Daniel Kroening, kroening@kroening.com
 #ifndef CPROVER_UTIL_ARITH_TOOLS_H
 #define CPROVER_UTIL_ARITH_TOOLS_H
 
+#include "deprecate.h"
```

Comment:
We could actually get rid of this include by removing `to_unsigned_integer`, which isn't used.